app/models/search.rb

```diff
@@ -138,6 +138,7 @@ def name!
     where!(
       "(concat(people.first_name, ' ', people.last_name) #{like} :full_name
        or (families.name #{like} :full_name)
+       or (people.alias) #{like} :full_name
```

Comment:
Yes, that should work the same (even if the nickname is 2 words)  Unless two people with the same first word in their 2 word nicknames exist.... I'll change it